numpy/lib/tests/test_arraypad.py

```diff
@@ -1,6 +1,6 @@
-"""Tests for the array padding functions.
+"""Tests for the array padding functions."""
```

Comment:
These changes all look cosmetic. I don't think any of them are really necessary are they?

Reply:
No, you're right. I'll keep try to keep the diff minimal in the future. :wink:

Reply:
Although, I would say they are appropriate in small PRs if they improve consistency. ;)